manifests/plugin/python.pp

```diff
@@ -38,7 +38,7 @@
     $ensure_real = 'absent'
   }
 
-  if $::operatingsystem == 'Fedora' {
+  if $::operatingsystem == 'Fedora' or $::operatingsystem == 'Amazon' {
```

Comment:
can you use the new facts syntax here? $facts['os']['name'] is probably correct.